components/menu-breadcrumbs-sd.tpl

```diff
@@ -3,9 +3,8 @@
 
 <ul class="menu menu-horizontal menu-public menu-breadcrumbs">
   {%- for listItem in breadcrumbsObj.itemListElement %}
-    {%- assign pageUrl = page.url | remove_first: "/" -%}
     <span class="menu-separator">/</span>
-    <li class="selected menu-item {% if pageUrl == listItem.item.id %} current{% endif %}">
+    <li class="selected menu-item {% if forloop.last %}current{% endif %}">
```

Comment:
It isn't a breaking error, but the `selected` class should be removed in this case for clarity, as the `current` class already states, which item is selected. (The `selected` class doesn't provide any additional styling rules either.)